### PR TITLE
github: fix settings

### PR DESCRIPTION
**Pull request by zimbatm**

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -123,15 +123,15 @@ branches:
       # Required. Require status checks to pass before merging. Set to null to disable
       required_status_checks:
         # Required. Require branches to be up to date before merging.
-        strict: true
+        strict: false
         # Required. The list of status checks to require in order to merge into this branch
-        contexts: [ "bors" ]
```
Comment (zimbatm) [inline .github/settings.yml:128]: I'm not sure what to put here

Comment (zowoq) [reply .github/settings.yml:128]: Could probably leave this as null for now and revisit it once CI has been stable for a bit.

File: .github/settings.yml [hunk 2]
```diff
@@ -123,15 +123,15 @@ branches:
...
+        contexts: []
       # Required. Enforce all configured restrictions for administrators. Set to true to enforce required status checks for repository administrators. Set to null to disable.
       enforce_admins: false
       # Disabled for bors to work
       required_linear_history: false
       # Required. Restrict who can push to this branch. Team and user restrictions are only available for organization-owned repositories. Set to null to disable.
       restrictions:
-        apps: [ "bors" ]
+        apps: []
```
Comment (zowoq) [inline .github/settings.yml:135]: I think mergify needs permissions like bors.

```suggestion
        apps: [ "mergify" ]
```

Comment (zimbatm) [reply .github/settings.yml:135]: This means that only mergify can push to main. Is that what we want?

File: .github/settings.yml [hunk 3]
```diff
@@ -123,15 +123,15 @@ branches:
...
         users: []
         teams: []
```
Comment (zowoq) [inline .github/settings.yml:137]:
```suggestion
        teams: [ "srvos" ]
```

**Pull request merged**